server.js

```diff
@@ -26,6 +26,7 @@ app.get('/view_all_projects', (req, res) => {
 
 app.get('/project_board/:id', async (req, res) => {
     const project = await Project.findByPk(req.params.id)
+    console.log(project)

```

Comment:
can remove it now?

Reply:
yes! I realised after i'd pushed, it's just a duplicate so I've removed from my version